### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for Windows, .deb, .rpm, Mac download links

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#24]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in EDGE.
2. Navigate to the Windows, .deb, .rpm, Mac download links available under the "Get started" Header
3. Observe the Localized Control Type property in Inspect

**_Same issue is also observed with the links:_**
[https://code.visualstudio.com/Download](https://code.visualstudio.com/Download)
https://code.visualstudio.com/insiders [Refer Screenshot]

**Actual Result:**
The localized control type property is buttons instead of links

**_Narrator Behaviour:_**
Windows Windows 7, 8, 10 button
.deb Debian, Ubuntu button
.rpm Red Hat, Fedora, CentOS button
Mac macOS 10.X button

**Expected Result:**
The localized Control Type property should be link instead of buttons as because in spite of providing the download link, clicking on the particular link redirects the user to some other page.

**User Impact:**
The visually impaired users will not be able to understand the exact purpose of the link

**Screenshots:**
![insiders link windows localized control type](https://cloud.githubusercontent.com/assets/24825441/21709306/00379282-d406-11e6-87fe-7040cd0a3d6d.png)
![localized control type of mac is button](https://cloud.githubusercontent.com/assets/24825441/21709304/002f8628-d406-11e6-906a-e961ef6cd6a9.png)
![localized control type of windows is button](https://cloud.githubusercontent.com/assets/24825441/21709305/002fc264-d406-11e6-8511-91a14a9c01a9.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/e5666cde8df5932957fc3dd1af31f88d27faaee2

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
